cime_config/testlist_allactive.xml

```diff
@@ -142,7 +133,7 @@
       <option name="wallclock"> 01:30:00 </option>
     </options>
   </test>
-  <test name="PFS" grid="f09_g17" compset="B1850" testmods="allactive/defaultio">
+  <test name="PFS" grid="f09_t232" compset="B1850" testmods="allactive/defaultio">
```

Comment:
Do we have and PFS tests on the BLT1850_v0c compset?   That would be preferred I think.

Reply:
Changed it to ne30pg3_t232 BLT1850_v0c

Reply:
great thanks